reports/report_details.py

```diff
@@ -239,8 +239,8 @@ def add_transnational_to_regions(regions):
     'ws_14': {
         '2015': {
             'name': '14',
-            'title': 'Position or occupation of news sources, by sex',
-            'desc': 'Breakdown of new sources by occupation and sex',
+            'title': 'Position or Occupation of News Subjects, by sex',
+            'desc': 'B.down by sex and occupation, show N &%',
```

Comment:
```suggestion
            'desc': 'B.down by sex and occupation',
```
Again, I think **_show N &%_** is a directive of what should be included in the chart rather than part of title (Please confirm with Jean).